### PR TITLE
Fixed NetworkInformationException for IPv4 disabled NICs.

### PR DESCRIPTION
**Pull request by Oskar-Armbruster**
Implemented fix for NetworkInformationException on IPv4 disabled NICs as suggested by @tmds (see issue #14).

### CODE REVIEW

File: src/Tmds.MDns/ServiceBrowser.cs [hunk 1]
```diff
@@ -250,6 +250,8 @@ private void CheckNetworkInterfaceStatuses(Dictionary<int, NetworkInterfaceHandl
                         continue;
                     }
 
+                    if (!networkInterface.Supports(NetworkInterfaceComponent.IPv4)) continue;
+
```
Comment (tmds) [inline src/Tmds.MDns/ServiceBrowser.cs]: Can you use the same style of the other checks?

File: src/Tmds.MDns/ServiceBrowser.cs [hunk 2]
```diff
@@ -149,7 +149,7 @@ internal void OnServiceRemoved(ServiceInfo service)
         }
 
         void OnNetworkInterfaceAdded(NetworkInterface networkInterface)
-        {            
+        {
```
Comment (tmds) [inline src/Tmds.MDns/ServiceBrowser.cs]: whitespace change?

File: src/Tmds.MDns/ServiceBrowser.cs (hunk 1 above)
Comment (Oskar-Armbruster) [reply src/Tmds.MDns/ServiceBrowser.cs]: Will do so!

File: src/Tmds.MDns/ServiceBrowser.cs (hunk 2 above)
Comment (Oskar-Armbruster) [reply src/Tmds.MDns/ServiceBrowser.cs]: This chage did not show up in my local diff viewer. Will try to omit it.

**Comment by tmds** [comment]
Thank you @Oskar-Armbruster !

**Comment by Oskar-Armbruster** [comment]
Thank YOU for the support and the great library!

**Comment by tmds** [comment]
Version 0.7.0 is on nuget.org that includes these changes.

**Pull request merged**